<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>지자체 설정 관리</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>원본대조필</string>
      </attribute>
      <layout class="QVBoxLayout" name="tab1_layout">
       <item>
        <widget class="QLabel" name="label_title">
         <property name="text">
          <string>지역별 원본대조필 자동 적용 설정</string>
         </property>
         <property name="styleSheet">
          <string>font-size: 14pt; font-weight: bold; margin-bottom: 10px;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="search_layout">
         <item>
          <widget class="QLabel" name="label_search">
           <property name="text">
            <string>지역명 검색:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_search">
           <property name="placeholderText">
            <string>지역명을 입력하세요 (초성 검색 가능)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_sealed">
         <column>
          <property name="text">
           <string>지역</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>원본대조필 적용</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="button_layout_1">
         <item>
          <widget class="QPushButton" name="btn_refresh">
           <property name="text">
            <string>새로고침</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_save">
           <property name="text">
            <string>저장</string>
           </property>
           <property name="styleSheet">
            <string>background-color: #4CAF50; color: white; font-weight: bold; padding: 5px 15px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_close">
           <property name="text">
            <string>닫기</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>보조금</string>
      </attribute>
      <layout class="QVBoxLayout" name="tab2_layout">
       <item>
        <widget class="QLabel" name="label_info">
         <property name="text">
          <string>보조금 관리 탭 (추후 구현 예정)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
